cmd/cluster/checkbanneduser.go

```diff
@@ -63,8 +64,16 @@ func CheckBannedUser(clusterID string) error {
 			fmt.Println("User banned due to export control compliance.\nPlease follow the steps detailed here: https://github.com/openshift/ops-sop/blob/master/v4/alerts/UpgradeConfigSyncFailureOver4HrSRE.md#user-banneddisabled-due-to-export-control-compliance .")
 			return nil
 		}
-		fmt.Println("Recommend sending service log with:")
-		fmt.Printf("osdctl servicelog post -t https://raw.githubusercontent.com/openshift/managed-notifications/master/ocm/cluster_owner_disabled.json %v\n", clusterID)
+
+		fmt.Println("Sending service log.")
```

Comment:
I'm not completely sure we would want the service logs to be sent fully automatically. We don't do that in any of our automation scripts AFAIK. 
Maybe adding a y/n confirmation to it would be better?

Reply:
As stated in the PR description, there will still be a prompt to send the service log. It will function exactly as if you called `osdctl servicelog post...` yourself.

Reply:
Ah, makes sense! Sorry for not reading ;)